### PR TITLE
Add new props repeat, disableSeek, hideForwardSkip

### PR DESCRIPTION
**Pull request by prettymuchbryce**
Hey! Neat project. I have added a few new options for my own needs, and I thought I would give you the option of merging them for others to use if you felt it was appropriate. I have one more change in mind, but it is unrelated so I will open a separate PR for it soon.

These are the options I added.

* `hideForwardSkip`: a boolean value that if true disables the forward skip button by hiding it from view. **false** by default.
* `disableSeek`: a boolean value that if true prevents seeking. **false** by default.
* `repeat`: a boolean value that if true continues playing from the beginning after the playlist has completed. **true** by default.

I have tested these changes locally without issue. I'm happy to change or remove any of the options as well if you don't want all of them.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -216,6 +225,7 @@ class AudioPlayer extends React.Component {
       displayedTime: 0
     }, () => {
       this.updateSource();
+      if (i === 0 && !this.props.repeat) shouldPlay = false;
```
Comment (benwiley4000) [inline src/index.js]: To be honest I would be fine with something like:
```javascript
const shouldPause = (
  (i === 0 && !this.props.repeat) ||
  (typeof shouldPlay === 'boolean' ? !shouldPlay : false)
);
```

**Comment by benwiley4000** [comment]
@prettymuchbryce Perhaps `cycle` instead of `repeat`?

**Comment by prettymuchbryce** [comment]
Thanks for your feedback! Good call on the immutable param. That's not something I normally do, but it's a good habit now that I think about it.

I've changed the option to `cycle`. I'm partial to using something like `repeatPlaylist` (making room for `repeatTrack` in the future), but if you still prefer cycle then this should do it.

**Comment by benwiley4000** [comment]
I expect the interface will need to be changed in the future if we include something like track repeats, but I'll stick with `cycle` for now. Moving forward I would be interested in thinking about controls for perhaps swapping between norepeat/playlist repeat/track repeat (as some sort of enumerable).

Thanks for the changes!

**Comment by benwiley4000** [comment]
@prettymuchbryce relevant: #53.

**Pull request merged**